lightning-js/ui-components
# fix(Solid): adds toggling between keyboards

## Description

Adds toggling between keyboard while also maintaining the prior positioning of the keyboard

## Changes

- adds keyboard input email story
- changes the structure of the formats prop to be `Record<string, Array<Array<string | Partial<KeyProps>>>>`
- refactors style

## Testing

- make sure keyboard renders as expected
- toggle between keyboards by going to an email story and clicking one of the special keys (`#@!`, `shift`, `áöû`). notice the changing of the keybaord
